build.gradle

```diff
@@ -46,7 +46,10 @@ dependencies {
     annotationProcessor 'org.springframework.boot:spring-boot-configuration-processor'
     annotationProcessor 'org.projectlombok:lombok'
     testImplementation 'org.springframework.boot:spring-boot-starter-test'
-    testImplementation 'org.springframework.security:spring-security-test'
+    testImplementation 'org.springframework.cloud:spring-cloud-starter-contract-stub-runner'
+    testImplementation "org.testcontainers:testcontainers"
+    testImplementation 'org.springframework.cloud:spring-cloud-starter-bootstrap'
+    testImplementation'org.testcontainers:mysql:1.16.2' //idc how autoversion that
```

Comment:
you probably can't as it's not covered by spring boot dependency management, so specifying exact version is OK in this case